iphone/Classes/TiUIWebView.m

```diff
@@ -540,6 +540,13 @@ -(void)setHandlePlatformUrl_:(id)arg
     willHandleUrl = [TiUtils boolValue:arg];
 }
 
+-(void)setUserAgent_:(id)args
+{
+    ENSURE_TYPE(args, NSString);
+    NSDictionary *dict = @{@"UserAgent":args};
```

Comment:
By passing null , it will create exception. And it will not reset to default user agent. Is this really required to give facility to reset default one ?